core/src/main/scala/io/treeverse/clients/SSTableReader.scala

```diff
@@ -18,12 +17,12 @@ private object local {
   }
 }
 
-class SSTableIterator[Proto <: Message](val it: SstFileReaderIterator, val messagePrototype: Proto) extends Iterator[Item[Proto]] with Closeable {
+class SSTableIterator[Proto <: GeneratedMessage](val it: SstFileReaderIterator, messagePrototype: GeneratedMessageCompanion[Proto]) extends Iterator[Item[Proto]] with Closeable {
```

Comment:
It's no longer a prototype, so perhaps
```suggestion
class SSTableIterator[Proto <: GeneratedMessage](val it: SstFileReaderIterator, companion: GeneratedMessageCompanion[Proto]) extends Iterator[Item[Proto]] with Closeable {
```
:question: